<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>658</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CRM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>1041</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_rechercheTri">
         <item>
          <widget class="QLCDNumber" name="lcdNumber">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="accessibleName">
            <string/>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="smallDecimalPoint">
            <bool>false</bool>
           </property>
           <property name="digitCount">
            <number>5</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
           <property name="value" stdset="0">
            <double>0.000000000000000</double>
           </property>
           <property name="intValue" stdset="0">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LineEditChercher">
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="BoutonAlpha">
           <property name="text">
            <string>A-Z</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">BoutonTri</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="BoutonDate">
           <property name="text">
            <string>Date</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">BoutonTri</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDateEdit" name="DateDebut">
         <property name="date">
          <date>
           <year>2023</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="DateFin">
         <property name="date">
          <date>
           <year>2023</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listViewAffichage">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_boutonsContact">
         <item>
          <widget class="QPushButton" name="BNouveau">
           <property name="text">
            <string>Nouveau</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BSupprimer">
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_afficherListe">
         <property name="text">
          <string>Ouvrir BDD</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1083</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Langue</string>
    </property>
    <addaction name="actionFran_ais"/>
    <addaction name="actionAnglais"/>
    <addaction name="actionArabe"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuLanguage"/>
  </widget>
  <action name="actionExport">
   <property name="text">
    <string>Export JSON</string>
   </property>
  </action>
  <action name="actionFran_ais">
   <property name="text">
    <string>Fran√ßais</string>
   </property>
  </action>
  <action name="actionAnglais">
   <property name="text">
    <string>Anglais</string>
   </property>
  </action>
  <action name="actionArabe">
   <property name="text">
    <string>Arabe</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="BoutonTri"/>
 </buttongroups>
</ui>
